courses/free-courses-en.md

```diff
@@ -1579,6 +1579,7 @@
 * [Web Basics](https://open.cs.uwaterloo.ca/web-basics/) - Centre for Education in Math and Computing (University of Waterloo)
 * [Web Development Course](https://syllabus.migracode.org/courses/introduction-3) - MigraCode Barcelona
 * [Web Development for Beginners - A Curriculum](https://github.com/microsoft/Web-Dev-For-Beginners) - Microsoft
+* [WEB DEVELOPMENT Tutorial By Coding Ninjas (In English)](https://www.youtube.com/playlist?list=PLrk5tgtnMN6TNuhUEf5-UgPxa1wu-l6kP) - Coding Ninjas
```

Comment:
no ALL CAPS

Reply:
Sorry for the mistake. I have made the necessary changes and updated the file.